pkg/test/builder/apiresource.go

```diff
@@ -36,9 +36,9 @@ func APIResourceList(version string, kinds []string) []*metav1.APIResourceList {
 func apiresources(version string, kinds []string) []metav1.APIResource {
 	apires := make([]metav1.APIResource, 0)
 	for _, kind := range kinds {
-		namespaced := false
+		namespaced := true
 		if strings.Contains(kind, "cluster") {
-			namespaced = true
+			namespaced = false
```

Comment:
😅  was it used anywhere?

Reply:
not sure, It did not make any difference anywhere